docs/examples/instances.yaml

```diff
@@ -1,22 +1,31 @@
-# https://docs.aws.amazon.com/emr/latest/APIReference/API_JobFlowInstancesConfig.html
+---
 Instances:
-  Ec2SubnetIds:
-  - subnet-1
-  - subnet-2
-  - subnet-3
-  KeepJobFlowAliveWhenNoSteps: True
-  Ec2KeyName: emr-key
-  # https://docs.aws.amazon.com/emr/latest/APIReference/API_InstanceFleetConfig.html
-  InstanceFleets:
-  - InstanceFleetType: MASTER
-    Name: master
-    TargetOnDemandCapacity: 1
-    InstanceTypeConfigs:
-    - InstanceType: m5.xlarge
-      WeightedCapacity: 1
-  - InstanceFleetType: CORE
-    Name: core
-    TargetOnDemandCapacity: 1
-    InstanceTypeConfigs:
-    - InstanceType: m5.xlarge
-      WeightedCapacity: 1
+  AdditionalMasterSecurityGroups:
+  - "$ADD_MASTER_SG"
+  AdditionalSlaveSecurityGroups:
+  - "$ADD_SLAVE_SG"
+  Ec2SubnetId: "$SUBNET_ID"
+  EmrManagedMasterSecurityGroup: "$MASTER_SG"
+  EmrManagedSlaveSecurityGroup: "$SLAVE_SG"
+  InstanceGroups:
```

Comment:
Could you change this back to `InstanceFleet` type config please, as that gives us some flexibility around AZ placement, use of Spot instances, etc.

Reply:
Fixed by https://github.com/dwp/emr-launcher/pull/2/commits/bb99a6938ed020c650423230b55d7120c251d1ab